lib/test/pages/BlockPage.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+/* jshint ignore:start */
+const BlockPage = function() {
```

Comment:
Why do you not use ES6 classes?

Reply:
see examples from official protractor site: https://github.com/angular/protractor/blob/master/docs/page-objects.md

Reply:
I see, probably they still support ES5. Ok, let’s keep it as is.